AISKU/src/Initialization.ts

```diff
@@ -30,22 +39,64 @@ const _ignoreUpdateSnippetProperties = [
  * @interface Snippet
  */
 export interface Snippet {
-    config: IConfiguration & Common.IConfig;
+    config: IConfiguration & IConfig;
     queue?: Array<() => void>;
     sv?: string;
     version?: number;
 }
 
-export interface IApplicationInsights extends Common.IAppInsights, IDependenciesPlugin, Common.IPropertiesPlugin {
+export interface IApplicationInsights extends IAppInsights, IDependenciesPlugin, IPropertiesPlugin {
     appInsights: ApplicationInsights;
     flush: (async?: boolean) => void;
     onunloadFlush: (async?: boolean) => void;
 };
 
+// Re-exposing the Common classes as Telemetry, the list was taken by reviewing the generated code for the build while using
+// the previous configuration :-
+// import * as Common from "@microsoft/applicationinsights-common"
+// export const Telemetry = Common;
+
 /**
  * Telemetry type classes, e.g. PageView, Exception, etc
  */
-export const Telemetry = Common;
+export const Telemetry = {
```

Comment:
See comment above lines 54..57 for why this
The other reason is that api-generator doesn't support the following mappings
import * as Common from "@microsoft/applicationinsights-common"